requirements.txt

```diff
@@ -1,10 +1,13 @@
-optuna==2.0.0
-Sphinx
+optuna==2.3.0
```

Comment:
Is this PR upgrading to Optuna 2.3.0? If so, let's change the title.

Reply:
Yes